README.md

```diff
@@ -1,12 +1,17 @@
 # [Shopify](https://www.shopify.com) JavaScript Buy SDK
 [![Circle CI](https://circleci.com/gh/Shopify/js-buy-sdk.png?circle-token=3be0ebe6fbb4841442b86678696947bd4b5456d7)](https://circleci.com/gh/Shopify/js-buy-sdk)
 
-The JS Buy SDK is a lightweight library that allows you to build ecommerce into any website. It is based on Shopify's API and provides the ability to retrieve products and collections from your shop, add products to a cart, and checkout.
+The JS Buy SDK is a lightweight library that allows you to build ecommerce into
+any website. It's based on Shopify's JSON API and provides the ability to
+retrieve products and collections from your shop, add products to a cart, and
+checkout.
 
-[Read documentation](http://shopify.github.io/js-buy-sdk/)
+If you're just getting started, [read the docs](http://shopify.github.io/js-buy-sdk/).
```

Comment:
I'd say _please_ read the docs personally, but maybe that's just me :stuck_out_tongue_winking_eye: 

:maple_leaf: